CalibTracker/SiStripESProducers/plugins/geom/SiStripRegionConnectivity.cc

```diff
@@ -45,7 +49,13 @@ SiStripRegionConnectivity::SiStripRegionConnectivity(const edm::ParameterSet& ps
   tTopoToken_ = cc.consumes();
 }
 
-SiStripRegionConnectivity::~SiStripRegionConnectivity() {}
+void SiStripRegionConnectivity::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.addUntracked<unsigned int>("EtaDivisions", 10);
```

Comment:
The configs for SiStripRegionConnectivity are either set or linked in several other parts in CMSSW, see https://github.com/search?q=repo%3Acms-sw%2Fcmssw%20SiStripRegionConnectivity&type=code , and everywhere the parameters are set as in your CalibTracker/SiStripESProducers/python/SiStripRegionConnectivity_cfi.py : why are you defining as default parameters in the `fillDescriptions` method some different ones?

Reply:
Good point...